lib/glimesh_web/templates/gct/edit_user_profile.html.eex

```diff
@@ -0,0 +1,151 @@
+<div class="container">
+    <div class="alert alert-danger mt-4 mb-1"><h2 class="mb-0 text-center"><%= gettext("WARNING YOU'RE CURRENTLY EDITING %{user}'s PROFILE SETTINGS", user: @user.username) %></h2></div>
+
```

Comment:
How can we not have this page duplicated since its the same view as the user profile?

Reply:
Was thinking of turning the card on that page into a component that is shared by both the user profile and the GCT side.